src/main/java/com/github/susom/database/Flavor.java

```diff
@@ -311,6 +319,11 @@ public String typeDate() {
       return "timestamp(3)";
     }
 
+    @Override
+    public String typeDbDate() {
+      return "date";
```

Comment:
Note that for Oracle, DATE includes time to second precision, and adjusts for daylight savings, which can sometimes shift things to the wrong day. Other databases might also do strange things, which is partly why I hadn't tried to figure out local dates yet... :)

Reply:
DATE, TIME WITH TIMEZONE, TIMESTAMP WITH TIMEZONE, TIME WITHOUT TIMEZONE, TIMESTAMP WITHOUT TIMEZONE, this is all perfectly clear in the SQL standard.  And it's not as if any of the major databases diverge from the standard.  I don't see the problem. 🤣